src/main/java/com/example/blog/domain/comment/controller/CommentController.java

```diff
@@ -0,0 +1,42 @@
+package com.example.blog.domain.comment.controller;
+
+import com.example.blog.domain.comment.domain.Comment;
+import com.example.blog.domain.comment.dto.CommentReq;
+import com.example.blog.domain.comment.dto.CommentRes;
+import com.example.blog.domain.comment.service.CommentService;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@Slf4j
+@AllArgsConstructor
+@RequestMapping("/comments")
+public class CommentController {
+    private final CommentService commentService;
+
+    @PostMapping
+    public ResponseEntity<CommentRes> createComment(@RequestBody CommentReq commentReq) {
+        Comment comment = commentService.createComment(commentReq.userId(), commentReq.postId(), commentReq.toEntity());
```

Comment:
컨트롤러 계층에서 엔티티를 선언하고 사용하는 것은 지양되고 있습니다! 컨트롤러에서 데이터의 전달은 Dto로만 이루어지는 것이 좋아요! 엔티티의 접근은 서비스계층에서 하는것을 추천드립니다 :)
```suggestion
        CommentResponse commentRes = commentService.createComment(commentReq);
```